Userland/Libraries/LibWeb/Page/Page.h

```diff
@@ -168,6 +176,9 @@ class Page : public Weakable<Page> {
     Optional<bool> m_pending_confirm_response;
     Optional<Optional<String>> m_pending_prompt_response;
 
+    PendingNonBlockingDialog m_pending_non_blocking_dialog { PendingNonBlockingDialog::None };
+    Optional<WeakPtr<HTML::HTMLInputElement>> m_pending_non_blocking_dialog_target;
```

Comment:
`Optional<WeakPtr<T>>` is a bit redundant. `WeakPtr<T>` already has a "nothing" state.